curriculum/challenges/english/25-front-end-development/lab-lunch-picker-program/66db529d37ad966480ebb633.md

```diff
@@ -49,20 +60,31 @@ You should define a function `addLunchToEnd`.
 assert.isFunction(addLunchToEnd);
 ```
 
-The function `addLunchToEnd` should take a single argument.
+The function `addLunchToEnd` should take 2 arguments.
 
 ```js
-assert.lengthOf(addLunchToEnd, 1);
+assert.lengthOf(addLunchToEnd, 2);
 ```
 
-`addLunchToEnd("Tacos")` should return the string `"Tacos added to the end of the lunch menu."`.
+`addLunchToEnd(lunches, "Tacos")` should log the string `"Tacos added to the end of the lunch menu."`.
 
 ```js
-assert.strictEqual(addLunchToEnd("Tacos"), "Tacos added to the end of the lunch menu.");
+const tempArr = [];
+const temp = console.log;
+try {
+  console.log = obj => tempArr.push(obj);
+  addLunchToEnd(lunches, "Pizza");
+  assert.strictEqual(tempArr[0], "Pizza added to the end of the lunch menu.");
+} finally {
+  console.log = temp;
+}
+```
+
+The function `addLunchToEnd` should return the correct array.
```

Comment:
```suggestion
`addLunchToEnd(["Pizza", "Tacos"], "Burger")` should return `["Pizza", "Tacos", "Burger"]`.
```
I would change the other hints to something like this so one can see the expected result.